Topics/Tech_Stacks/ReactNative.md

```diff
@@ -1,30 +1,51 @@
+
+Meanwhile, [EXPO](https://docs.expo.dev) is a open-source platform built around React Native. It contains a series of useful tools for developing a React Native app. This includes convenient commands to create and test your app universally on iOS, Android and web without additional configuration. Hence, it is highly recommended to use Expo if you are beginning the development of a mobile app. Furthermore, even existing React Native apps can have the `expo` package installed. <br>
+
+With Expo's [EAS Update](https://docs.expo.dev/eas-update/introduction/) and [EAS Build](https://docs.expo.dev/build/introduction/) features, development builds can be launched instantly, and your app can be tested on mobile devices or mobile device simulators. Official builds can also be launched efficiently to share among your team, or even make its way to AppStore and Google Play. Additionally, Expo offers a [web dashboard](https://expo.dev) with concise UI that aids the management of projects and their deployment. You can sign up for a free Expo account and manage your projects on this dashboard. The [Expo-CLI](https://docs.expo.dev/more/expo-cli/) is the command-line tool that connects all these tools together, and even helps to securely manage your project's dependencies.<br>
 
    1. Set up
       1. Installation
          1. Install Node.js and npm (Node Package Manager) on your computer. You can download and install Node.js from the official website: https://nodejs.org/en/
-            #Note: Suggested to download the LTS version because it is generally more stable than the Current or Latest version.
-         2. Install expo go on your ios device (https://apps.apple.com/us/app/expo-go/id982107779) or android device (https://play.google.com/store/apps/details?id=host.exp.exponent&hl=en_CA&gl=US&pli=1)
+            #Note: Suggested to download the LTS version (or even numbered versions npm) because it is generally more stable. Use `npm -v` and `Node -v` to check your installed version.
+         2. Install Expo Go on your [iOS device](https://apps.apple.com/us/app/expo-go/id982107779) or [Android device](https://play.google.com/store/apps/details?id=host.exp.exponent&hl=en_CA&gl=US&pli=1) via App Store and Google Play Store
          3. Download VS Code Extensions
             1. ES7+ React/Redux/React-Native snippets (Extension ID: dsznajder.es7-react-js-snippets)
             2. React Native Tools (Extension ID: msjsdiag.vscode-react-native)
       2. Command
-         1. Install expo
+         1. Install Expo CLI (command line tool)
             `npm install -g expo-cli`
```

Comment:
The global package expo-cli has to be deprecated

Reply:
Fixed